src/components/Summary/AlterationSummary.vue

```diff
@@ -354,6 +355,10 @@ export default class AlterationSummary extends Mixins(
   }
 }
 
+#name-translation {
+  width: 100%;
```

Comment:
Instead of this, would removing the v-row around your new component work?

Reply:
yes indeed, change made